src/main/features/core/notificationManager.ts

```diff
@@ -15,27 +17,32 @@ export default class NotificationManager extends Feature {
   public register() {
     // Track changed
     this.subscribe<PlayingTrack>(['player', 'playingTrack'], ({ currentState }) => {
-      if (!this.win || (this.win && this.win.isFocused())) {
-        return;
-      }
-
       const {
         player: { playingTrack },
         config: {
           app: { showTrackChangeNotification }
         }
       } = currentState;
 
-      if (playingTrack && showTrackChangeNotification) {
+      if (playingTrack) {
         const trackId = playingTrack.id;
         const track = getTrackEntity(trackId)(currentState);
 
-        if (track) {
-          this.sendToWebContents(EVENTS.APP.SEND_NOTIFICATION, {
-            title: track.title,
-            message: `${track.user && track.user.username ? track.user.username : ''}`,
-            image: SC.getImageUrl(track, IMAGE_SIZES.SMALL)
-          });
+        if (settings.get('app.logTrackChange')) {
+          fs.writeFileSync('/tmp/auryo_status.log', `${track?.title}\n${track?.user?.username}`);
+        }
+
+        if (!this.win || (this.win && this.win.isFocused())) {
+          return;
+        }
+        if (showTrackChangeNotification) {
+          if (track) {
```

Comment:
I would try and combine these if statements. Perhaps you could dedicate a seperate variable for the isFocused. It's more readable.
```javascript
const isFocused = !this.win || (this.win && this.win.isFocused());

if(track && showTrackChangeNotification && !isFocused)
...
```